File: JavaScript/1528. Shuffle String.js
/**
 * @param {string} s
 * @param {number[]} indices
 * @return {string}
 */
var restoreString = function(s, indices) {
    var a = [], b = 0;
    for(var i = 0;i < s.length;i++){
        a[indices[i]] = s[i];
    }
    return a.join("");
};

File: Java/1528. Shuffle String.java
class Solution {
    public String restoreString(String s, int[] indices) {
        String[] r = new String[s.length()];
        for(int i = 0; i< s.length(); i++){
            r[indices[i]] = String.valueOf(s.charAt(i));
        }
        return String.join("", r);
    }
}
